### PR TITLE
Add csv checking feature in clean function

### PR DESCRIPTION
**Pull request by YoonhoKim0527**

### CODE REVIEW

File: repository/src/lib.rs [hunk 1]
```diff
@@ -217,6 +217,23 @@ impl<T: RawRepository> DistributedRepository<T> {
             {
                 self.raw.delete_branch(branch.to_string()).await?;
             }
+
+            // Only accept csv acceptable branches
```
Comment (fakedev9999) [inline repository/src/lib.rs]:
```suggestion
            // Delete branches with invalid commit sequence
```

Comment (YoonhoKim0527) [reply repository/src/lib.rs]: Fixed.

File: repository/src/lib.rs [hunk 2]
```diff
@@ -217,6 +217,23 @@ impl<T: RawRepository> DistributedRepository<T> {
...
+
+            // Delete branches with invalid commit sequence
+            let reserved_state = self.get_reserved_state().await?;
+            let commits =
+                read_commits(self, finalized_branch_commit_hash, branch_commit_hash).await?;
+            let last_header = self.get_last_finalized_block_header().await?;
+            let mut verifier =
+                CommitSequenceVerifier::new(last_header.clone(), reserved_state.clone())
+                    .map_err(|e| eyre!("failed to create a commit sequence verifier: {}", e))?;
+            for (commit, _hash) in commits.iter() {
```
Comment (fakedev9999) [inline repository/src/lib.rs]:
```suggestion
            for (commit, _) in commits.iter() {
```

Comment (YoonhoKim0527) [reply repository/src/lib.rs]: Fixed.

**Comment by fakedev9999** [comment]
LGTM

**Pull request merged**